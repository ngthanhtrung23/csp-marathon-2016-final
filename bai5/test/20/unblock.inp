1.2344
1.23..
100356
778.56
.98..A
.9BBBA
