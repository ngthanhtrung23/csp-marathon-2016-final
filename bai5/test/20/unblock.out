24
3 D 1
4 L 1
6 U 2
A U 1
B R 1
8 D 1
7 R 1
1 D 3
0 L 1
2 D 1
4 L 3
2 U 1
0 R 1
1 U 1
3 U 1
5 U 2
7 R 2
8 U 1
9 U 1
B L 3
A D 1
7 R 1
3 D 3
0 R 3

